src/requirements/data/majors/math.ts

```diff
@@ -0,0 +1,396 @@
+        ),
+        perSlotMinCount: [3, 2],
+        slotNames: ['Mathematical Biology Courses', '3000+ MATH Course'],
+      },
+      Physics: {
+        description:
+          '5 courses from (i) and (ii): ' +
+          '(i) At least one MATH course numbered 3000 or above, ' +
+          '(ii) At least three physics courses that make significant use of advanced mathematics.',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          [
+            'PHYS 3316',
+            'PHYS 3317',
+            'PHYS 3318',
+            'PHYS 3327',
+            'PHYS 4230',
+            'PHYS 4443',
+            'PHYS 4444',
+            'PHYS 4445',
+            'PHYS 4454',
+            'PHYS 4481',
+            'PHYS 4488',
+            'AEP 4340',
+            'AEP 4400',
+          ],
+          [
+            'PHYS 3316',
+            'PHYS 3317',
+            'PHYS 3318',
+            'PHYS 3327',
+            'PHYS 4230',
+            'PHYS 4443',
+            'PHYS 4444',
+            'PHYS 4445',
+            'PHYS 4454',
+            'PHYS 4481',
+            'PHYS 4488',
+            'AEP 4340',
+            'AEP 4400',
+            'MATH 3***',
+            'MATH 4***',
+            'MATH 5***',
+            'MATH 6***',
+            'MATH 7***',
+          ]
+        ),
+        perSlotMinCount: [1, 3, 1],
+        slotNames: ['3000+ MATH Course', 'Physics Courses', 'MATH or Physics Course'],
+      },
+      'Operations Research': {
+        description:
+          '5 courses from (i) and (ii): ' +
+          '(i) At least one MATH course numbered 3000 or above, ' +
+          '(ii) At least three courses in ORIE in which the primary focus involves mathematical techniques.',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          [
+            'ORIE 3300',
+            'ORIE 3310',
+            'ORIE 3500',
+            'ORIE 3510',
+            'ORIE 4150',
+            'ORIE 4350',
+            'ORIE 4520',
+            'ORIE 4600',
+            'ORIE 4630',
+            'ORIE 4740',
+            'ORIE 5600',
+            'ORIE 5610',
+            'ORIE 5640',
+          ],
+          [
+            'ORIE 3300',
+            'ORIE 3310',
+            'ORIE 3500',
+            'ORIE 3510',
+            'ORIE 4150',
+            'ORIE 4350',
+            'ORIE 4520',
+            'ORIE 4600',
+            'ORIE 4630',
+            'ORIE 4740',
+            'ORIE 5600',
+            'ORIE 5610',
+            'ORIE 5640',
+            'MATH 3***',
+            'MATH 4***',
+            'MATH 5***',
+            'MATH 6***',
+            'MATH 7***',
+          ]
+        ),
+        perSlotMinCount: [1, 3, 1],
+        slotNames: ['3000+ MATH Course', 'ORIE Courses', 'MATH or ORIE Course'],
```

Comment:
Can you say ORIE Course instead of ORIE Courses? Try to make the slot names singular throughout.